frontend/src/routes/tree/+page.svelte

```diff
@@ -455,7 +455,7 @@
               {#if results}
                 <button
                   class="p-1 px-3 bg-blue-500/40 rounded disabled:bg-blue-900/40 mr-2"
-                  on:click={() => openTrade(searchJewel, searchConqueror, searchResults.raw)}
+                  on:click={() => openTrade(searchJewel, searchConqueror, searchResults.raw, true)}
```

Comment:
I would still like to retain the logic, that if it can, it should generate for all conquerors.